app/src/main/java/com/sunlightlabs/android/congress/fragments/RollListFragment.java

```diff
@@ -291,18 +287,16 @@ public View getView(int position, View view, ViewGroup parent) {
 			holder.question.setText(Utils.truncate(roll.question, 200));
 			holder.result.setText(resultFor(roll));
 
-            // vote views can use an optional 'details' pane for more info
-            if (roll.nomination != null && roll.nomination.nominees != null) {
+            if (roll.bill != null) {
                 holder.details.setVisibility(View.VISIBLE);
-                holder.detailsText.setText(Nomination.nomineesFor(roll.nomination));
-            } else if (roll.amendment != null && roll.amendment.amends_bill_id != null) {
-                holder.details.setVisibility(View.VISIBLE);
-                String text = "Amendment to " + Bill.formatCode(roll.amendment.amends_bill_id) + ": " + Amendment.description(roll.amendment);
-                holder.detailsText.setText(Utils.truncate(text, 300));
 
-            } else if (roll.bill != null) {
-                holder.details.setVisibility(View.VISIBLE);
-                holder.detailsText.setText(Bill.formatCode(roll.bill_id) + ": " + roll.bill.short_title);
+                String title;
+                if (roll.bill.official_title != null)
+                    title = Utils.truncate(roll.bill.official_title, 200);
+                else
+                    title = "(untitled)";
```

Comment:
I also fixed the embedded bill display to use a (truncated if necessary) `official_title` instead of `short_title`, which was showing `null` in the UI when a vote happened on a bill without a short title. I think this was an accidental change I made when messing with titles in #690.